### DISCUSSION TITLE
Path to upgrade from Golf to Hotel release when installed to internal SSD

### DISCUSSION THREAD

**Discussion by brandonb927** [kg4vdk/arcos-linux-modules#42] [It's Not In The Manual...Promise!] [answered]
Hello! I don't know where best to ask this but I have a machine that I installed the Golf release to on the internal SSD, but wanted to upgrade to the new Hotel release. I did it the hard way initially as I had no data on the drive by wiping Golf and installing Hotel, but wanted to know if there will be an upgrade path published between future versions? Normally in other distros I would just run the OS equivalent of `apt dist-upgrade`, so not sure what the path is here with a liveUSB-driven OS.

Brandon VE7TZB

**Comment by kg4vdk** [comment, marked as answer]
Hey Brandon, you've asked in the right place! Hopefully you didn't run into any issues getting Golf wiped and Hotel installed. As you may have guessed, since this is a live OS, there isn't the possibility of using something like `dist-upgrade`. The method you used is the upgrade path...re-write the new ISO to your boot media (USB or SSD).

If you had any user modules/scripts, offline maps, etc. saved elsewhere, you should be able to copy them over to Hotel and continue using them. I *believe* the only module with a significant change is Pat. In Hotel, Pat is configured to accommodate VARA modems. My personal method is to start "fresh" and the only thing I bring over is offline maps and any manuals or other documents that I keep in persistent storage.

Also, while Hotel is live (21.3.0), it was a quiet release and I am already working on Hotel 21.3.1. Just a few minor tweaks (mostly to clarify some of the language of the notifications. If you run into any issues or confusion, please let me know! Things will certainly need to be fixed, or documented in the Field Manual (which should be placed in your `$HOME` directory, once your modules are up-to-date).

Mike

**Reply by brandonb927** [reply]
Thanks Mike, appreciate the response. I'll stay tuned for the point release!
